scripts/ci/install-rust.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -ex
+
+
+RUST_LATEST_VERSION=1.52.1
+# For security, we specify a particular rustup version and a SHA256 hash, computed
+# ourselves and hardcoded here. When updating `RUSTUP_LATEST_VERSION`:
+#   1. Download the new rustup script from https://github.com/rust-lang/rustup/releases.
+#   2. Audit the script and changes to it. You might want to grep for strange URLs...
+#   3. Update `OUR_RUSTUP_SHA` with the result of running `sha256sum rustup-init.sh`.
+RUSTUP_LATEST_VERSION=1.24.2
+OUR_RUSTUP_SHA="40229562d4fa60e102646644e473575bae22ff56c3a706898a47d7241c9c031e"
+
+
+# Download rustup script
+curl --proto '=https' --tlsv1.2 -sSf \
+  https://raw.githubusercontent.com/rust-lang/rustup/${RUSTUP_LATEST_VERSION}/rustup-init.sh -O && \
```

Comment:
Apparently it's not breaking the script, but are these trailing `&& \`s doing anything? AFAIK the blank lines after each one will get registered as the next command and turn into a no-op

Reply:
Hm. I'm not sure. Let me look into this...

Reply:
Hm... http://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Lists

> A list is a sequence of one or more pipelines separated by one of the operators ‘;’, ‘&’, ‘&&’, or ‘||’, and optionally terminated by one of ‘;’, ‘&’, or a newline.
>
> Of these list operators, ‘&&’ and ‘||’ have equal precedence, followed by ‘;’ and ‘&’, which have equal precedence.
>
> A sequence of one or more newlines may appear in a list to delimit commands, equivalent to a semicolon.

Reply:
My understanding has been that

```sh
command && commnd

command \
&& command

command && \
command
```

are all equivalent.

The question is whether

```sh
command && \

command
```

is equivalent to those as well, yes?

Reply:
I'm happy to remove these.

`set -e` should provide us with the desired behavior: exiting the script after any failed command.